src/main.js

```diff
@@ -3,22 +3,29 @@ var userInputBody = document.querySelector('#userCardBody');
 var starredIdeaBtn = document.querySelector('.starred-ideas-btn');
 var saveIdeaBtn = document.querySelector('.save-idea-btn');
 var savedIdeaCards = [];
-
+var ideaForm = document.querySelector('.user-idea-form')
 var savedCardsGrid = document.querySelector('.saved-cards-grid')
 
 
-
+ideaForm.addEventListener('keydown', checkUserInputs);
```

Comment:
Lets try 'keyup' instead of 'keydown' and try to utilize the disable button instead of what we have here
lets take a look at what we covered in small group sessions today!